test/format-test.cc

```diff
@@ -2533,9 +2540,9 @@ TEST(FormatTest, EmphasisNonHeaderOnly) {
 TEST(FormatTest, CharTraitsIsNotAmbiguous) {
   // Test that we don't inject internal names into the std namespace.
   using namespace std;
-  char_traits<char>::char_type c;
+  FMT_MAYBE_UNUSED char_traits<char>::char_type c;
```

Comment:
I suggest just doing

```
(void)c;
```
and removing the `FMT_MAYBE_UNUSED` macro because it is not very portable (requires C++17).